app/components/UI/TransactionElement/utils.js

```diff
@@ -163,10 +178,16 @@ function getTokenTransfer(args) {
 		};
 	}
 
+	let transactionType;
+	if (renderFullAddress(from) === selectedAddress) transactionType = TRANSACTION_TYPES.SENT_TOKEN;
+	else transactionType = TRANSACTION_TYPES.RECEIVED_TOKEN;
+
```

Comment:
I think using let and if/else conditionals makes sense if there's greater than two possibilities, but for cases like this we can use a ternary and `const`:

(I've also added some destructuring for the `TRANSACTION_TYPES`, but that can likely go at the `import`)

```suggestion
	const { SENT_TOKEN, RECEIVED_TOKEN } = TRANSACTION_TYPES
	const transactionType = (renderFullAddress(from) === selectedAddress)
		? SENT_TOKEN
		: RECEIVED_TOKEN
```